CPAC/info.py

```diff
@@ -189,15 +189,15 @@ def get_cpac_gitversion():
     "pathvalidate==2.5.2",
     "patsy==0.5.0",
     "prov==1.5.2",
-    "psutil==5.4.6",
+    "psutil==5.6.6",
     "pybids==0.15.1",
     "pygraphviz==1.3.1",
     "PyPEER",
     "python-dateutil==2.7.3",
-    "pyyaml==5.3",
+    "pyyaml==5.4",
     "scikit-learn==0.22.1",
-    "scipy==1.6.0",
-    "sdcflows==2.0.2",
+    "scipy==1.6.3",
```

Comment:
Worth noting: we don't seem to have consistent versions of `numpy` across the environments... possibly other libraries as well, but that is an impactful one that we should be careful with.

Reply:
Yep. We're well overdue for [a dependencies upgrade](https://github.com/FCP-INDI/C-PAC/issues/1673), and for the variant images we need to put together some process for how often we update our variants to match the corresponding packages